pkg/dsl/task/task_http.go

```diff
@@ -0,0 +1,95 @@
+package task
+
+import (
+	"context"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	"cuelang.org/go/cue"
+)
+
+func init() {
+	Register("http", newHTTPCmd)
```

Comment:
Can you write some test cases for this task?

Reply:
done